### DISCUSSION TITLE
应用列表无法显示部分系统应用

### DISCUSSION THREAD

**Discussion by RemiDre** [MetaCubeX/ClashMetaForAndroid#409] [General]
我是用的是vivo的手机，clash使用的是"仅允许已选择的应用"走代理。有一个系统app叫teeservice需要走代理，不然fcm断联后不会重连，导致无法接收fcm推送。
我已经去掉了"过滤系统应用"的对勾，但应用列表中还是找不到teeservice，使用flclash是可以正常找到teeservice的

**Comment by sacc-leo** [comment]
i have met this problem too. Some apps can not found in the installed applications, such as google play store.
but v2rayng can.
maybe clash meta depends on some old lib for getting installed apps list？

**Reply by RemiDre** [reply]
The Google Play Store is generally a system app. Did you uncheck the "Filter system apps" option? My issue is that the package filtering logic has a problem - some apps that don't require internet permissions are being incorrectly filtered out by ClashMeta.
The code is in AccessControlActivity.kt file,line 121 [AccessControlActivity.kt](https://github.com/MetaCubeX/ClashMetaForAndroid/blob/main/app/src/main/java/com/github/kr328/clash/AccessControlActivity.kt#L121)

**Reply by RemiDre** [reply]
But I still recommend you use flclash,it has fewer bugs and a great UI design.

**Comment by mustang651** [comment]
Same problem here. "Filter system apps" is unchecked of course.
Tested Devices & Results:
Samsung Galaxy S24 Ultra & S25 Ultra (Android 16, One UI): Google Gemini app is installed but missing from the CMFA list.
Samsung Galaxy S22 (Android 16, One UI): Google Gemini is visible, BUT a custom test APK (downloaded from GitHub) is missing.
Samsung Galaxy Tab S7 (Android 12, One UI): The same custom test APK and Gemini are visible.

P.S.
FlClash doesn't support SOCKS proxy authentication properly.
